optimizely/vercel-examples
# update deploy url and user id generation mechanism

### Description

1. Deploy url was pointing to the fork. Updated that to point to the main repository.
2. user id generation was being done using a naive 6 digit random number generator. Changed it to use `uuid` library instead.

### Best Way to Test

1. Try the `Deploy` button and verify the repository is being cloned from the main repo and not the fork.
2. Run the application and check edgeworker logs. The user id should be uuid instead of a 6 digit integer id.

### Type of Change

- [x ] Bug fix (non-breaking change that fixes an issue)
- [ x] New feature in existing example

### New Example Checklist

- [x ] 🚀 Link to deployment URL on Vercel
- [ x] 📱 Is it responsive? Are mobile and tablets considered?
- [ x] 📚 `README.md` preview link in PR description (branch)
- [x ] ⚙️ Secrets have instructions for how to set up in the readme
